core/src/docs/rfcs/2884_merge_range_read_into_read.md

```diff
@@ -0,0 +1,79 @@
+- Proposal Name: `merge_range_read_into_read`
+- Start Date: 2023-08-20
+- RFC PR: [apache/incubator-opendal#2884](https://github.com/apache/incubator-opendal/pull/2884)
+- Tracking Issue: [apache/incubator-opendal#2811](https://github.com/apache/incubator-opendal/issues/2811)
+
+# Summary
+
+Merge the `range_read` API into `read` by deleting the `op.range_reader(path, range)` and `op.range_read(path, range)` method.
+
+# Motivation
+
+Currently OpenDAL has separate `appender` and `writer` APIs:
```

Comment:
Seems the content is not filled.

Reply:
I just want to PIN PR number in current commit, Now I have change these content. Please take at look it.